code/modules/clothing/glasses/hud.dm

```diff
@@ -64,9 +64,29 @@
 	icon_state = "healthhud"
 	deactive_state = "degoggles_med"
 	flags_armor_protection = 0
-	toggleable = 1
+	toggleable = TRUE
+	hud_type = DATA_HUD_MEDICAL_ADVANCED
+	actions_types = list(/datum/action/item_action/toggle)
+
+/obj/item/clothing/glasses/hud/medgoggles
+	name = "\improper HealthMate ballistic goggles"
+	desc = "Standard issue TGMC goggles. With an internal HealthMate HUD projector."
+	icon_state = "medgoggles"
+	item_state = "medgoggles"
+	deactive_state = "degoggles_medgoggles"
+	toggleable = TRUE
 	hud_type = DATA_HUD_MEDICAL_ADVANCED
 	actions_types = list(/datum/action/item_action/toggle)
+	soft_armor = list("melee" = 40, "bullet" = 40, "laser" = 0, "energy" = 15, "bomb" = 35, "bio" = 10, "rad" = 10, "fire" = 30, "acid" = 30)
+	flags_equip_slot = ITEM_SLOT_EYES
+	goggles = TRUE
+
+/obj/item/clothing/glasses/hud/medgoggles/prescription
+	name = "\improper HealthMate prescription ballistic goggles"
+	desc = "Standard issue TGMC goggles. With an internal HealthMate HUD projector and prescription lenses."
+	icon_state = "medgoggles"
+	item_state = "medgoggles"
```

Comment:
subtype, doenst need icon_state and item_state, parent already has

Reply:
Pretty sure I followed the way standard prescription goggles were set, so I’ll snip that when I get a chance.

Reply:
Fixed this and did the same for regular prescription goggles in the new commit.